cmd/ebs-snapshotter/main.go

```diff
@@ -150,18 +148,8 @@ func loadVolumeSnapshotConfig(volumeSnapshotConfigFile string) *models.VolumeSna
 }
 
 func initialiseHTTPServer(port int) {
-	router := mux.NewRouter()
-
-	router.NewRoute().PathPrefix("/__/").
-		Methods(http.MethodGet).
-		Handler(getOpHandler())
-
-	router.NewRoute().Path("/_/metrics").
-		Methods(http.MethodGet).
-		Handler(promhttp.Handler())
-
-	loggingHandler := handlers.LoggingHandler(os.Stdout, router)
-	http.Handle("/", loggingHandler)
+	http.Handle("/__/", getOpHandler())
```

Comment:
or this really... you can just pass the handler generated from the op lib to a http server, example inbound

Reply:
not the best but the first one i found 

```
log.Fatal(http.ListenAndServe(fmt.Sprintf(":%d", port), getOpHandler()))
```